### PR TITLE
QOL improvements

### PR DESCRIPTION
**Pull request by Gadsee**
Cleaning

# Description

This PR holds some clean-up QOL improvements which were a few lines of code of change.

1. The bot no longer levels after level 8. I've never seen it reach level 9, so this should save some gold.
2. The button that appears if you win a game is now recognized as a valid "Exit Now" button. The bot wouldn't break on victory, but if it does reach first place, it waits the full 25 seconds before continuing.
3. We now log the storage path, closing #157 
    a. I could not figure out a better way to require UAC elevation, so we'll just have to keep asking users to run as admin. At least now this issue is also logged in the console.
4. Refactors the resource_path method to a) get the _MEIPASS variable in a more "gentle" manner and b) remove it entirely, since it didn't do anything anymore.

# Notes

Had to move out a few methods in main() since we started having too many, but I think that's a good change.

### CODE REVIEW

File: captures/buttons/continue.png
Comment (Kyrluckechuck) [inline captures/buttons/continue.png:1]: nit: Could be worth cropping this down just a tinsy bit further to not contain colours outside the button -- I don't _think_ this should be a big enough issue, but something to consider

File: tft.py [hunk 1]
```diff
@@ -788,41 +838,17 @@ def main():
     logger.info(f"Bot will only display messages at severity level {log_level}.")
     logger.info(f"Bot will {'' if config.forfeit_early() else 'NOT '}surrender early.")
 
+    absolute_storage_path = os.path.abspath(storage_path)
+    if not os.access(absolute_storage_path, os.W_OK):
+        logger.error(f"We do not have write access into {absolute_storage_path}. Try running the bot as administrator.")
+    else:
+        logger.info(f"All of the configuration can be found in '{absolute_storage_path}'.")
```
Comment (Kyrluckechuck) [inline tft.py:845]: Should also mention that logs can be found here, too

File: tft.py [hunk 2]
```diff
@@ -735,19 +736,68 @@ def re_add_non_debug_logger(log_level: str) -> None:
...
+
+
 @logger.catch
 def main():
     """Entrypoint function to initialize most of the code.
 
     Parses command line arguments, sets up console settings, logging, and kicks of the main bot loop.
     """
+    # This enables us to always use relative paths and avoid unicode issues in paths.
+    os.chdir(getattr(sys, "_MEIPASS", os.path.abspath(".")))
```
Comment (Kyrluckechuck) [inline tft.py:792]: Comment: Neat 👌

File: captures/buttons/continue.png
Comment (Gadsee) [reply captures/buttons/continue.png:1]: Yea, we have a default tolerance of 20%, but you're right. I also wanted to suggest cropping down the trait captures, but that can wait until set 9 where we need to re-take most of them anyway.

File: tft.py [hunk 3]
```diff
@@ -735,19 +736,68 @@ def re_add_non_debug_logger(log_level: str) -> None:
...
+                    f"Your version: {versions[0]}\n"
+                    f"Current version: {versions[1]}"
+                ),
+                buttons=["Download", "Ignore"],
+            )
+            == "Download"
+        ):
+            logger.info(f"{releases_url} should open in your browser now")
+            webbrowser.open(f"{releases_url}", new=0, autoraise=True)
+            sys.exit(1)
```
[the remark this answers is not in the document]
Comment (Gadsee) [reply tft.py:764]: I agree, this should be exit 0. Changing it while we're modifying this piece of code.

**Comment by Gadsee** [comment]
> Would it make sense to also add the starting of League (and any other sub-programs) as non-administrator to this?

Actually, it is better to run League as admin, according to their [official support page](https://support-leagueoflegends.riotgames.com/hc/en-us/articles/204134104-Setting-up-your-Permissions). I don't have a strong opinion on adding it, just wanted to bring this up for discussion first before doing so.

**Review by Kyrluckechuck — APPROVED** [review]

**Pull request merged**